src/Plugin/tmgmt/Translator/ThebigwordTranslator.php

```diff
@@ -226,7 +227,9 @@ public function requestJobItemsTranslation(array $job_items) {
         throw new TMGMTException($message);
       }
 
-      $job->submitted('Job has been successfully submitted for translation.');
+      if ($job->isContinuous()) {
+        $job_item->active('Requested translation to the continuous translator.', [], 'debug');
```

Comment:
You don't request to, you request from.

Or for a continuous item.

newTranslationProject(JobInterface $job) always uses continuous job id for ProjectReference and PurchaseOrderNumber. Are we sure that is OK?

* separate branch for required_by refactoring
* separate: CMS User Email is set twice, second can be removed.